### ISSUE TITLE
Unable to clean cache for some apps

### ISSUE THREAD

**Issue by SohanRay** [d4rken-org/sdmaid-se#1359]
**Describe the bug**
When using the app to automatically clear cache for all apps , including system apps using accessibility service,its unable to clear cache for Chrome, Phonepe UPI, Shadow fight 4 game in Android.

**To Reproduce**
Use appcleaner in the sd maid app and keep setting to clear cache for system apps also.

**Expected behavior**
should clear cache for all apps 

**Smartphone (please complete the following information):**
 - Device: OnePlus 12R
 - OS: Android 14
 - Root status - not rooted 
 - SD Maid 2/SE version -  v1.1.2-rc0

**Debug log**
If you can reproduce the bug, record the event in [a debug log](https://github.com/d4rken-org/sdmaid-se/wiki/Bugs#debug-log).

**Screenshots/Video**
If applicable, add screenshots or a video if it helps to explain your problem.

**Additional context**

**Comment by d4rken** [comment]
Are you able to clear the caches manually? Without root SD Maid only presses the "Clear cache" button, if clicking that makes has no effect, then the issue is with the system's clear cache routine.

Other common causes: The app are recreating the files fast, the files are in use, or the files did get deleted, but the system doesn't update the cache size information.

**Comment by SohanRay** [comment]
Hi,

Yes I am able to clear the cache manually. Like by going to the settings.

**Comment by d4rken** [comment]
Please provide a debug log of this:
https://github.com/d4rken-org/sdmaid-se/wiki/Bugs#debug-log

If you don't clean all apps, but just prses the problematic ones in SD Maid, does it also not work?

I wonder if the screen looks different, or maybe the system takes longer to calculate sizes :thinking:

**Comment by SohanRay** [comment]
Here's the log you asked for.

**Comment by d4rken** [comment]
It's unfortunately not possible to send attachments to GitHub via Email.

**Comment by SohanRay** [comment]
Here's the log.
[eu.darken.sdmse_logfile_1723020579486.zip](https://github.com/user-attachments/files/16523045/eu.darken.sdmse_logfile_1723020579486.zip)

**Comment by SohanRay** [comment]
Hi,

Added the log zip file manually on GitHub

**Comment by d4rken** [comment]
Thanks for the logfile.

This looks like a system issue, the system is not returning screen information for the apps that are skipped:

```java
2024-08-07T08:50:23.423Z  V/SDMSE:AutomationService: Waiting for windowRoot...
2024-08-07T08:50:23.676Z  V/SDMSE:Automation:Service: Providing windowRoot: null
2024-08-07T08:50:23.676Z  V/SDMSE:AutomationService: Waiting for windowRoot...
2024-08-07T08:50:23.930Z  V/SDMSE:Automation:Service: Providing windowRoot: null
2024-08-07T08:50:23.930Z  V/SDMSE:AutomationService: Waiting for windowRoot...
2024-08-07T08:50:24.185Z  V/SDMSE:Automation:Service: Providing windowRoot: null
2024-08-07T08:50:24.186Z  V/SDMSE:AutomationService: Waiting for windowRoot...
2024-08-07T08:50:24.439Z  V/SDMSE:Automation:Service: Providing windowRoot: null
(...)
```

Are you able to clear these apps if you just select them individually in SD Maid? Maybe the system is getting overloaded :thinking:

**Comment by SohanRay** [comment]
Nope. Even if I select them individually, I am unable to clear the cache using SD maid. Is it system issue or system compatibility issue? Because I was using the old SD maid (paid for that one too), until just 2 days ago, and it worked fine for all apps!

**Comment by SohanRay** [comment]
Confirmed . Sd maid 1 works without issues but sd maid 2 fails to clear cache for a number of applications .

**Comment by d4rken** [comment]
I don't see a reason why it would work for SDM1 and not SDM2, the code is not the same but similar.

Also there is no app specific handling in AppCleaner. So if it works for one app, but not the other app, there must be a difference between them in the system.

I'm currently out of ideas, do you spot any difference between the apps that work and that don't?

**Comment by SohanRay** [comment]
Hi,
Well, its unable to delete cache for like 50% of the apps . And whenever
its unable to delete cache for some app , when the overlay appears saying I
cannot use the device currently , in the background for those apps , I have
noticed that sd maid 2 is unable to navigate to storage after reaching app
info page to be able tap on clear app cache.

**Comment by d4rken** [comment]
Duplicate of #1016
